ext/mysql2/client.h

```diff
@@ -65,9 +65,11 @@ void rb_mysql_client_set_active_thread(VALUE self);
 
 void init_mysql2_client(void);
 void decr_mysql2_client(mysql_client_wrapper *wrapper);
+void set_server_query_flags(MYSQL *client, VALUE result);
```

Comment:
This function is only called from within client.c, so you can make it static and leave it out of the header file.